include/RAJA/RAJA.hxx

```diff
@@ -65,39 +65,39 @@
 
 #include "RAJA/internal/defines.hxx"
 
-#include "RAJA/int_datatypes.hxx"
-#include "RAJA/real_datatypes.hxx"
+#include "RAJA/internal/int_datatypes.hxx"
+#include "RAJA/internal/real_datatypes.hxx"
 
-#include "RAJA/operators.hxx"
+#include "RAJA/internal/operators.hxx"
```

Comment:
Non-rhetorical question: should operators be internal? I could see these being things a well-architected RAJA code might be exposed to. I could go either way